src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -103,7 +101,8 @@
     public DoubleSolenoid climbAngle;
 
     // index
-    public WPI_TalonFX indexMotor;
+    public WPI_TalonFX firstIndexMotor, secondIndexMotor;
+    public MultiplexedColorSensor firstIndexColorSensor, secondIndexColorSensor;
 
```

Comment:
Recommend naming these based on their location (feeder or ingest) instead of just first & second

Reply:
Ditto for the motors, if they are in different places (left / right)?